app/services/discord/clear_roles_service.rb

```diff
@@ -16,13 +16,11 @@ def execute
           roles: []
         )
       rescue Discordrb::Errors::UnknownMember
-        Rails.logger.error "Unknown member #{@user.discord_user_id}"
-        @user.update!(discord_user_id: nil)
```

Comment:
I think we should have this logic in-place. 
Can't we do something like this 
```rb
User.find_by(discord_user_id: @discord_user_id).update!(discord_user_id: nil)
```
This will ensure that we will clear incorrect data from our db if its present;